manifests/params.pp

```diff
@@ -271,4 +271,16 @@
   if $init_style == undef {
     fail('Unsupported OS')
   }
+
+  # Autodetect whether we should use camptocamp/systemd
```

Comment:
I don't like this because I don't it will work as you expect

Reply:
How come?
It worked last time I used it.
https://github.com/cpitman/puppet-database_schema/pull/4